modules/post/multi/gather/lastpass_creds.rb

```diff
@@ -726,13 +725,12 @@ def decrypt_vault_password(key, encrypted_data)
     return nil if key.blank? || encrypted_data.blank?
 
     if encrypted_data[0] == "!" # Apply CBC
-      decipher = OpenSSL::Cipher.new("AES-256-CBC")
+      decipher = OpenSSL::Cipher.new("AES-256-CBC").decrypt
       decipher.iv = encrypted_data[1, 16] # Discard !
```

Comment:
Apparently setting the `iv` before calling `decrypt` is fine. This could be left as it is - but I decided for consistency to align with the other uses in the code base.

I didn't test the module, but wired up a code snippet to test it out instead

<details>

<summary>Code snippet</summary>

```
Verified with:
require 'openssl'

def decrypt_vault_password(key, encrypted_data)
  # return nil if key.blank? || encrypted_data.blank?

  if encrypted_data[0] == "!" # Apply CBC
    decipher = OpenSSL::Cipher.new("AES-256-CBC")
    decipher.iv = encrypted_data[1, 16] # Discard !
    encrypted_data = encrypted_data[17..-1]
  else # Apply ECB
    decipher = OpenSSL::Cipher.new("AES-256-ECB")
  end
  decipher.decrypt
  decipher.key = [key].pack "H*"

  begin
    return decipher.update(encrypted_data) + decipher.final
  rescue OpenSSL::Cipher::CipherError
    vprint_error "Vault password could not be decrypted with key #{key}"
    return nil
  end
end

def encrypt_vault_password_cbc(plaintext)
  iv = OpenSSL::Digest::MD5.digest("hello world")
  key = "A" * 32

  cipher = OpenSSL::Cipher.new("AES-256-CBC")
  cipher.encrypt
  cipher.iv = iv
  cipher.key = key

  encrypted_deets = ""
  encrypted_deets << cipher.update(plaintext)
  encrypted_deets << cipher.final
  encrypted_data = "!#{iv}#{encrypted_deets}"

  [key.unpack1("H*"), encrypted_data]
end

def encrypt_vault_password_ecb(plaintext)
  key = "A" * 32

  cipher = OpenSSL::Cipher.new("AES-256-ECB")
  cipher.encrypt
  cipher.key = key

  encrypted_deets = ""
  encrypted_deets << cipher.update(plaintext)
  encrypted_deets << cipher.final
  encrypted_data = "#{encrypted_deets}"

  [key.unpack1("H*"), encrypted_data]
end

key, encrypted_data = encrypt_vault_password_cbc("hunter2")
puts "Unencrypted cbc result: #{decrypt_vault_password(key, encrypted_data)}"

key, encrypted_data = encrypt_vault_password_ecb("hunter2")
puts "Unencrypted ebc result: #{decrypt_vault_password(key, encrypted_data)}"
```

</details>